packages/console/src/cloud/components/Broadcast/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from 'react-i18next';
+
+import { logtoBlogLink } from '@/cloud/constants';
+import TextLink from '@/components/TextLink';
+
+import * as styles from './index.module.scss';
+
+const Broadcast = () => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <div className={styles.broadcast}>
+      <span>{t('cloud.broadcast')}</span>
+      <TextLink href={logtoBlogLink} target="_blank" rel="noopener" className={styles.link}>
```

Comment:
nit: the blog link is trustable? No need to set the rel as noopener?

Reply:
updated.